jetty9/src/main/docker/50-jetty.bash

```diff
@@ -0,0 +1,53 @@
+# Configure the environment and CMD for the Jetty Container
```

Comment:
I'm keen to keep this as a single PR as there area few overlapping concerns.   But I agree that we need to test this script.   I'll add test harness in some form shortly.